chart/tests/test_flower.py

```diff
@@ -22,12 +22,21 @@
 from tests.helm_template_generator import render_chart
 
 
-class FlowerAuthorizationTest(unittest.TestCase):
+class FlowerTest(unittest.TestCase):
+    def test_should_create_flower_by_celery_executor(self):
+        docs = render_chart(
+            values={"executor": "CeleryExecutor", "flower": {"enabled": True}},
+            show_only=["templates/flower/flower-deployment.yaml"],
+        )
+
+        assert "RELEASE-NAME-flower" == jmespath.search("metadata.name", docs[0])
+        assert "flower" == jmespath.search("spec.template.spec.containers[0].name", docs[0])
+
     def test_should_create_flower_deployment_with_authorization(self):
         docs = render_chart(
             values={
                 "executor": "CeleryExecutor",
-                "flower": {"username": "flower", "password": "fl0w3r"},
+                "flower": {"enabled": True, "username": "flower", "password": "fl0w3r"},
```

Comment:
perhaps it makes sense to not specify "enabled" here, to test what happens by default

Reply:
This test just test create flower with authorization, I will test separately without "enabled".